tests/run_tests.sh

```diff
@@ -8,6 +8,21 @@ fi
 PACKAGES_PATH=${1}
 SCRIPT_DIR=$(cd $(dirname $0); pwd)
 TMP_DIR_NAME="tmp"
+PYTHON_BIN=${PYTHON_BIN:-python3}
+
+# check if PYTHON_BIN binary is availble
+if ! command -v ${PYTHON_BIN} > /dev/null; then
+    echo "Error: Cannot find ${PYTHON_BIN} binary."
+    exit 1
+fi
+python=$(command -v ${PYTHON_BIN})
```

Comment:
Please address review comments as I mentioned in `src/gen_module.sh` to this file.

Reply:
see comment above, renamed